app/views/LocationTracking.js

```diff
@@ -291,6 +293,14 @@ class LocationTracking extends Component {
     }
     return BackgroundImage;
   }
```

Comment:
My understanding was that this was only an issue on iOS12.0.  I know in my testing on iOS 11 it wasn't a problem.  Please verify the specific iOS version needed.

Reply:
Okay, I will try on ios 11. I haven't tried on ios 11. If it is an issue with ios12.0 only then I will change the condition.

Reply:
I have checked the condition for 12.0 and 12.0.1 only. Please verify it.